helm_deploy/values-prod.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Per environment values which override defaults in hmpps-incentives-api/values.yaml
+
+generic-service:
+  replicaCount: 4
+
+  ingress:
+    host: incentives-api.hmpps.service.justice.gov.uk
+    contextColour: green
+
+  env:
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
```

Comment:
shouldn't need this variable - it is the default in the `values.yaml` anyway

Reply:
👍